config_template.yml

```
@@ -0,0 +1,40 @@
+### CONNECTION TO PERUN RPC ###
+rpc:
+  perun_url: "https://example.com"
+  perun_user: "example"
+  perun_password: "password"
+
+database:
+  connection_string: "mongodb://admin:test@127.0.0.1:27017/admin"
+  database_name: "admin"
+  collection_name: "test"
+
+### MAPPING OF CLIENT PROPERTIES TO PERUN ATTRIBUTES ###
+attributes:
+  client_id: "urn:perun:facility:attribute-def:def:OIDCClientID"
+  client_secret: "urn:perun:facility:attribute-def:def:OIDCClientSecret"
+  name: "urn:perun:facility:attribute-def:def:serviceName"
+  description: "urn:perun:facility:attribute-def:def:serviceDescription"
+  redirect_uris: "urn:perun:facility:attribute-def:def:OIDCRedirectURIs"
+  privacy_policy: "urn:perun:facility:attribute-def:def:privacyPolicyURL"
+  contacts:
+    - "urn:perun:facility:attribute-def:def:administratorContact"
+  allowed_scopes: "urn:perun:facility:attribute-def:def:requiredScopes"
+  grant_types: "urn:perun:facility:attribute-def:def:OIDCGrantTypes"
+  code_challenge_type: "urn:perun:facility:attribute-def:def:OIDCCodeChallengeType"
+  introspection: "urn:perun:facility:attribute-def:def:OIDCAllowIntrospection"
+  post_logout_redirect_uris: "urn:perun:facility:attribute-def:def:OIDCPostLogoutRedirectURIs"
+  issue_refresh_tokens: "urn:perun:facility:attribute-def:def:OIDCIssueRefreshTokens"
+  proxy_identifier: "urn:perun:facility:attribute-def:def:proxyIdentifiers"
+  master_proxy_identifier: "urn:perun:facility:attribute-def:def:masterProxyIdentifier"
+  is_test_sp: "urn:perun:facility:attribute-def:def:isTestSp"
+  is_oidc: "urn:perun:facility:attribute-def:def:isOidc"
+  manager_groups_id: "urn:perun:facility:attribute-def:def:managersGroupId"
+  home_page_uris:
+    - "urn:perun:facility:attribute-def:def:informationURL"
+  rp_type: "urn:perun:facility:attribute-def:def:rpType"
+
+rp_type_attr_value: "satosa"
+proxy_identifier_value: "proxyId"
+
+encryption_key: "secret"
```

```suggestion
encryption_key: "secret"

static_attributes:
  subject_type: public
  application_type: web
  token_endpoint_auth_method: none
```

We will need to set some global (static) attributes which cannot be edited in SP reg app, but SATOSA needs them to be set. This could be the config, just key:value pairs, which would be added to the client settings (before values from Perun, so that if the same attribute is here and in Perun, the value from Perun overrides the static value).